### PR TITLE
Moving mne

### PR DESCRIPTION
**Pull request by brianjohannes**
Removed MNE related methods from EegData and ErpData. I instead put them as helper functions in examples/mne. I turned an old broken py script into an example that plots in MNE using data from an XDF source, which I think will be useful for a few people in our lab. I removed examples from this folder that don't work anymore because EEG data is stored inside private variables in the EegData object. 

The example which plots ERP data as MNE Epochs probably shouldn't work, but it does because ErpData.erp_windows_processed is not private (it probably should be).

Working with these examples raised some questions for me about how we want to have EEG data accessible from EegData or alternatively how we want it spit out at the end.

TLDR: MNE is out of the way now, it is still broken, but probably not worth fully fixing until we decide how we want to make EEG data available for saving.

### CODE REVIEW

**Comment by gregbci** [comment]
Looks good.  Note example still need mne installed.
